src/containers/Transactions/simpleTab.scss

```diff
@@ -28,20 +28,20 @@ $subdued-color: $gray-400;
           .amount-localized {
             display: block;
           }
-        }
 
-        .currency {
-          display: block;
-          margin: -1px 0px -16px;
-          color: $subdued-color;
-          font-size: 11px;
-          text-align: right;
-          @include medium;
-
-          .account {
-            font-size: inherit;
-            line-height: inherit;
-            vertical-align: bottom;
+          .currency {
```

Comment:
There are places that .currency is not inside and amount in the unrefactored transactions like PaymentChannelCreate

```
        <div className="value">
          {amt}
          <Currency currency={amount.currency} amount={amount.amount} />
        </div>
 ```

Reply:
I know, I figured those were technically incorrect and needed to be refactored anyways, so it didn't matter.
If you want, I can refactor that in this PR/in a separate PR so that's not the case anymore.